app/models/host/base.rb

```diff
@@ -1,7 +1,7 @@
 module Host
   class Base < ApplicationRecord
     KERNEL_RELEASE_FACTS = [ 'kernelrelease', 'ansible_kernel', 'kernel::release' ]
-    UPTIME_FACTS = [ 'system_uptime::seconds', 'ansible_uptime_seconds', 'uptime_seconds', 'proc_stat::btime' ]
+    UPTIME_FACTS = [ 'system_uptime::seconds', 'ansible_uptime_seconds', 'uptime_seconds' ]
```

Comment:
Is this still used?

Reply:
These are still uptime facts which can be useful, similar to kernel release fact, now these are really just uptime facts, but if you prefer I can drop.

Reply:
If we don't use this somewhere, I'd prefer to drop it.